### ISSUE TITLE
TS2300 [ERROR]: Duplicate identifier 'Location'

### ISSUE THREAD

**Issue by erikbrinkman** [yj01jung/domtype#1]
When I try to import this module I get a type error that seems to stem from `Location` being declared twice, e.g.

```
deno run https://deno.land/x/domtype@1.0.3/mod.ts
```

produces

```
Check https://deno.land/x/domtype@1.0.3/mod.ts
error: TS2300 [ERROR]: Duplicate identifier 'Location'.
declare class Location {
              ~~~~~~~~
    at asset:///lib.deno.window.d.ts:58:15

    'Location' was also declared here.
    interface Location {
              ~~~~~~~~
        at https://deno.land/x/domtype@1.0.3/domtype.d.ts:11739:11    and here.
    declare var Location: {
                ~~~~~~~~
        at https://deno.land/x/domtype@1.0.3/domtype.d.ts:11813:13

TS2300 [ERROR]: Duplicate identifier 'Location'.
interface Location {
          ~~~~~~~~
    at https://deno.land/x/domtype@1.0.3/domtype.d.ts:11739:11

    'Location' was also declared here.
    declare class Location {
                  ~~~~~~~~
        at asset:///lib.deno.window.d.ts:58:15

TS2300 [ERROR]: Duplicate identifier 'Location'.
declare var Location: {
            ~~~~~~~~
    at https://deno.land/x/domtype@1.0.3/domtype.d.ts:11813:13

    'Location' was also declared here.
    declare class Location {
                  ~~~~~~~~
        at asset:///lib.deno.window.d.ts:58:15

Found 3 errors.
```

```
$ deno --version
deno 1.7.0 (release, x86_64-unknown-linux-gnu)
v8 8.9.255.3
typescript 4.1.3
```

**Comment by pcdinh** [comment]
I got the same issue when importing the module in Deno 1.7.5

**Comment by yj01jung** [comment]
Location issue was fixed in 1.0.4
